pandas/core/config.py

```diff
@@ -388,12 +390,14 @@ class option_context(object):
 
     """
 
-    def __init__(self, *args):
-        if not (len(args) % 2 == 0 and len(args) >= 2):
+    def __init__(self, *args, options={}):
```

Comment:
I don't think a signature allowing keyword only arguments after varargs is supported in Py27, so may be a compat issue? See [PEP 3102](https://www.python.org/dev/peps/pep-3102/)

Reply:
I see. I didn't actually realize that was a compatibility issue, but judging from the fact that all the 2.7 tests failed, I'd say you're right.